testassets/InteropTestsClient/InteropClient.cs

```diff
@@ -298,11 +316,82 @@ private async Task RunTestCaseAsync(IChannelWrapper channel, ClientOptions optio
                 case "server_compressed_streaming":
                     await RunServerCompressedStreamingAsync(client);
                     break;
+                case "grpcweb_unary":
```

Comment:
For InteropTestsClient and InteropTestsWebsite, I'd like to only add "official" interop tests that have a spec.
I know this is a convenient place to put the grpc-web test, but we probably don't want to mix official and unofficial interop tests.
If you think moving the tests elsewhere would be too difficult, can we at least add a note where these test are coming from and that they are temporary and commit to replacing the grpc-web tests with the official ones once the spec exists?

Reply:
I'll add a note. If/when grpc-web interop tests become more established then we could move them to their own server/client.